modules/webbrowser/src/processors/processorcefsynchronizer.cpp

```diff
@@ -64,6 +64,7 @@ bool ProcessorCefSynchronizer::OnQuery(CefRefPtr<CefBrowser> browser, CefRefPtr<
     try {
         auto command = j.at("command").get<std::string>();
         auto subscribeProgressCommand = std::string("processor.progress.subscribe");
+        auto unsubscribeProgressCommand = std::string("processor.progress.unsubscribe");
```

Comment:
const?

Reply:
could use constexpr string_view for these string literals...

Reply:
Fixed